src/azure/pci-dss-3.2.1/rules/pci-dss-3.2.1-monitoring-check-5.ts

```diff
@@ -0,0 +1,169 @@
+    bash -c 'curl -X PUT -H "Authorization: Bearer $1" -H "Content-Type:application/json" https://management.azure.com/subscriptions/$0/resourceGroups/<Resource_Group_ToCreate_Alert_In>/providers/microsoft.insights/activityLogAlerts/<Unique_Alert_Name>?api-version=2017-04-01 -d@"input.json"'
+  
+  Where input.json contains the Request body JSON data as mentioned below.
+  
+    {
+      "location": "Global",
+      "tags": {},
+      "properties": {
+      "scopes": [
+        "/subscriptions/<Subscription_ID>"
+      ],
+      "enabled": true,
+      "condition": {
+        "allOf": [
+        {
+          "containsAny": null,
+          "equals": "Administrative",
+          "field": "category"
+        },
+        {
+          "containsAny": null,
+          "equals": "Microsoft.Network/networkSecurityGroups/securityRules/write",
+          "field": "operationName"
+        }
+        ]
+      },
+      "actions": {
+        "actionGroups": [
+        {
+          "actionGroupId": "/subscriptions/<Subscription_ID>/resourceGroups/<Resource_Group_For_Alert_Group>/providers/microsoft.insights/actionGroups/<Alert_Group>",
+          "webhookProperties": null
+        }
+        ]
+      },
+      }
+    }
+
+  Configurable Parameters for command line:
+
+    <Resource_Group_To Create_Alert_In>
+    <Unique_Alert_Name>
+
+  Configurable Parameters for input.json:
+
+    <Subscription_ID> in scopes
+    <Subscription_ID> in actionGroupId
+    <Resource_Group_For_Alert_Group> in actionGroupId
+    <Alert_Group> in actionGroupId`,
+  
+  references: [
+    'https://azure.microsoft.com/en-us/updates/classic-alerting-monitoring-retirement',
+    'https://docs.microsoft.com/en-in/azure/azure-monitor/platform/alerts-activity-log',
+    'https://docs.microsoft.com/en-in/rest/api/monitor/activitylogalerts/createorupdate',
+    'https://docs.microsoft.com/en-in/rest/api/monitor/activitylogalerts/listbysubscriptionid',
+    'https://docs.microsoft.com/en-us/azure/security/benchmarks/security-controls-v2-logging-threat-detection#lt-4-enable-logging-for-azure-resources',
+  ],  
+  gql: `{
+    queryazureSubscription {
+      id
+      __typename
+      activityLogAlerts {
+        enabled
+        condition {
+          allOf {
+            field
+            equals
+          }
+        }
+      }
+    }
+  }`,
+  resource: 'queryazureSubscription[*]',
+  severity: 'medium',
+  conditions: {
+    path: '@.activityLogAlerts',
+    array_any: {
+      and: [
+        {
+          path: '[*].enabled',
+          equal: true,
+        },
+        {
+          path: '[*].condition.allOf',
+          array_any: {
+            and: [
+              {
+                path: '[*].field',
+                equal: 'operationName',
+              },
+              {
+                path: '[*].equals',
+                equal: 'microsoft.network/networksecuritygroups/securityrules/write',
```

Comment:
I'm seeing that the check-5 and the check-6 are the same. The check-5 is equivalent to the rule '`azure-cis-1.3.1-5.2.3`' the operation name should be equal to this: '`microsoft.network/networksecuritygroups/write`'
And check-6 would be fine and is equivalent to the rule '`azure-cis-1.3.1-5.2.5`'

Reply:
@m-pizarro good catch, Thanks! The check-5 is really off. Fixed the rule and its unit test.

Reply:
great!